web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -494,6 +522,152 @@ function ViewModeSwitch({
   );
 }
 
+export const IncludedResourcesSelector = ({
+  onChange,
+  includedResourceMode,
+}: {
+  onChange: (value: IncludedResourceMode) => void;
+  includedResourceMode: IncludedResourceMode;
+}) => {
+  const options: { value: IncludedResourceMode; label: string }[] = [
+    {
+      value: 'accessible',
+      label: 'Accessible',
+    },
+    {
+      value: 'requestable',
+      label: 'Requestable',
+    },
+  ];
+
+  const [availability, setAvailability] = useState<IncludedResourceMode[]>(
+    includedResourceMode === 'all'
+      ? ['requestable', 'accessible']
+      : [includedResourceMode]
+  );
+
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleOpen = event => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const onCancel = () => {
+    if (includeRequestable && includedResourceMode === 'all') {
```

Comment:
can we move line 568 (where we `const includeRequestable ....`) above this function somewhere (it probably works as is, but kinda confusing, b/c i initially didn't know where this variable was coming from)